### PR TITLE
[ingest] commits from GH Enterprise

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: bin/ingest-csv [hunk 1]
```diff
@@ -14,7 +14,6 @@ end
 if filename.nil?
   $stderr.puts "Usage: #{$PROGRAM_NAME} <csv file>"
   $stderr.puts "Usage: #{$PROGRAM_NAME} <csv file> <image directory> <object type>"
-  $stderr.puts "Usage: #{$PROGRAM_NAME} <csv file> <image directory>"
```
Comment (dunn) [inline bin/ingest-csv]: Doesn't work as of 6133eacaa65a91a55fdc7118c8c38bc7756f432c.

Comment (jcoyne) [inline bin/ingest-csv]: Did you include the `type` column in the csv?

Comment (jcoyne) [inline bin/ingest-csv]: I just tested this and it does work with two arguments.

Comment (dunn) [inline bin/ingest-csv]: Oh, so the updated script just won't work with old metadata?

Comment (jcoyne) [inline bin/ingest-csv]: You can use the old data files if you also pass the class as the third argument.

**Comment by jcoyne** [comment]
Closing in favor of #565

**Pull request closed without merging**